dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```
@@ -2030,10 +2042,10 @@
       "type": "object",
       "description": "Supported parameters for a Conversational PII detection and redaction task.",
       "properties": {
-        "piiCategories": {
+        "inclusionList": {
           "$ref": "#/definitions/ConversationPIICategories"
         },
-        "includeAudioRedaction": {
+        "includeAudioRedactionTiming": {
```

I will repeat my comment from the reviewing session here so it doesn't get lost. Did you consider a simpler name, e.g.
```suggestion
        "redactAudioTiming": {
```